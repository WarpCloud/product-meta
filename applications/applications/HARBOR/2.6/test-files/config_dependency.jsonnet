{
  application_type: "HARBOR",
  application_name: "harbor1",
  application_version: "2.6",
  user_config: {},
}
