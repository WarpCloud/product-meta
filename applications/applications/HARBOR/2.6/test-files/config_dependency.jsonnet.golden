{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "harbor1",
            "application_type": "HARBOR"
         },
         "configs": {
            "harbor_cpu_limit": 2,
            "harbor_cpu_request": 0.10000000000000001,
            "harbor_memory_limit": 4,
            "harbor_memory_request": 1
         },
         "instance_settings": { },
         "moduleName": "harbor",
         "name": "harbor1-harbor",
         "version": "2.6"
      }
   ],
   "instance_settings": { },
   "version": ""
}
